### ISSUE TITLE
Typo in error message

### ISSUE THREAD

**Issue by mattgarrish** [IDPF/epubcheck#546]
Should be "does not match"  in the following:

ERROR(RSC-005): .\WCAG-ch1.epub/EPUB/xhtml/toc.xhtml(10,17): Error while parsing file 'The heading rank h1 does match the current nesting level (2).'.

**Comment by rdeltour** [comment]
Fixed locally (PR pending).
